README.md

```diff
@@ -114,11 +90,6 @@ instrument .NET libraries and applications.
 - [AWS App Mesh](./service-mesh/appmesh): An example E-Commerce application with
   deployment and configuration files for App Mesh on ECS.
 
-### AWS Lambda Functions
-
-- [AWS Lambda](./aws-lambda): Examples for instrumenting spans for AWS Lambda written in Java, Python, Node, Go
```

Comment:
Do we have any `splunk-otel-lambda` examples? @kubawach

Reply:
Yep, in the opentelemetry-tracing / opentelemetry-lambda. I took a look at @iNikem changes and I'm OK with them.